packages/core/vitest-integration.setup.ts

```diff
@@ -1,28 +1,67 @@
-import { beforeEach } from "vitest"
+import { beforeEach, beforeAll, afterEach } from "vitest"
 import { PostgreSqlContainer, type StartedPostgreSqlContainer } from "@testcontainers/postgresql"
-import { createMigrator, createKysely } from "@dotkomonline/db"
-import { createEnvironment } from "@dotkomonline/env"
+import { createMigrator, createKysely, Database } from "@dotkomonline/db"
+import { Environment, createEnvironment } from "@dotkomonline/env"
+import { exec } from "child_process"
+import { Kysely } from "kysely"
+import util from "util"
+
+const promiseExec = util.promisify(exec)
+
+const password = "local"
+const username = "local"
+const database = "main"
 
 const containers: StartedPostgreSqlContainer[] = []
 
-beforeEach(async () => {
-  const container = await new PostgreSqlContainer("public.ecr.aws/z5h0l8j6/dotkom/pgx-ulid:0.1.3")
+let container: StartedPostgreSqlContainer
+let kysely: Kysely<Database>
+let env: Environment
+
+beforeAll(async () => {
+  container = await new PostgreSqlContainer("public.ecr.aws/z5h0l8j6/dotkom/pgx-ulid:0.1.3")
     .withExposedPorts(5432)
-    .withUsername("local")
-    .withPassword("local")
-    .withDatabase("main")
+    .withUsername(username)
+    .withPassword(password)
+    .withDatabase(database)
     .start()
+
+  containers.push(container)
   process.env.DATABASE_URL = container.getConnectionUri()
-  const env = createEnvironment()
-  const kysely = createKysely(env)
+}, 30000)
+
+beforeEach(async () => {
+  env = createEnvironment()
+  kysely = createKysely(env)
```

Comment:
Each test should probably be run within a transaction for isolation?

This is out-of-the-box in [django](https://docs.djangoproject.com/en/5.0/topics/testing/tools/#transactiontestcase)

Reply:
I'm not sure about this. Could you explain some more on what you mean by isolation?

With this setup the database is deleted and recreated between each test.